### PR TITLE
yt-dlp: add page

### PR DESCRIPTION
**Pull request by n14s**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/yt-dlp.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+> A youtube-dl fork with additional features and fixes.
+> Download videos from YouTube and other websites.
+> More information: <https://github.com/yt-dlp/yt-dlp>
+
+- Download a video or playlist (with the default options from command below):
+`yt-dlp "https://www.youtube.com/watch?v=oHg5SJYRHA0"`
+
+- Download best format that contains video, and if it doesn't already have an audio stream, merge it with best audio-only format (Default)
+`yt-dlp -f "bv*+ba/b "https://www.youtube.com/watch?v=oHg5SJYRHA0"`
```
Comment (navarroaxel) [inline pages/common/yt-dlp.md:11]: We prefer long arguments if available
```suggestion
`yt-dlp --format "{{bv*+ba/b}}" "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
```

File: pages/common/yt-dlp.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download YouTube playlist videos in separate directory indexed by video order in a playlist
+`yt-dlp -o "%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s" "https://www.youtube.com/playlist?list=PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re"`
```
Comment (navarroaxel) [inline pages/common/yt-dlp.md:14]:
```suggestion
`yt-dlp --output "{{%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{https://www.youtube.com/playlist?list=PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re}}"`
```

File: pages/common/yt-dlp.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# yt-dlp
+
+> A youtube-dl fork with additional features and fixes.
+> Download videos from YouTube and other websites.
+> More information: <https://github.com/yt-dlp/yt-dlp>
```
Comment (navarroaxel) [inline pages/common/yt-dlp.md:5]:
```suggestion
> More information: <https://github.com/yt-dlp/yt-dlp>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/yt-dlp.md (hunk 1 above)
Comment (n14s) [reply pages/common/yt-dlp.md:11]: Missed that, thanks alot!

File: pages/common/yt-dlp.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# yt-dlp
+
+> A youtube-dl fork with additional features and fixes.
+> Download videos from YouTube and other websites.
+> More information: <https://github.com/yt-dlp/yt-dlp>.
+
+- Download a video or playlist (with the default options from command below):
+`yt-dlp "https://www.youtube.com/watch?v=oHg5SJYRHA0"`
```
Comment (blueskyson) [inline pages/common/yt-dlp.md:8]:
```suggestion
- Download a video or playlist (with the default options from command below):

`yt-dlp "https://www.youtube.com/watch?v=oHg5SJYRHA0"`
```

File: pages/common/yt-dlp.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download best format that contains video, and if it doesn't already have an audio stream, merge it with best audio-only format (Default)
+`yt-dlp --format "{{bv*+ba/b}}" "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
```
Comment (blueskyson) [inline pages/common/yt-dlp.md:11]:
```suggestion
- Download best format that contains video, and if it doesn't already have an audio stream, merge it with best audio-only format (Default):

`yt-dlp --format "{{bv*+ba/b}}" "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
```

File: pages/common/yt-dlp.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download YouTube playlist videos in separate directory indexed by video order in a playlist
+`yt-dlp --output "{{%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{https://www.youtube.com/playlist?list=PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re}}"`
```
Comment (blueskyson) [inline pages/common/yt-dlp.md:14]:
```suggestion
- Download YouTube playlist videos in separate directory indexed by video order in a playlist:

`yt-dlp --output "{{%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{https://www.youtube.com/playlist?list=PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re}}"`
```

File: pages/common/yt-dlp.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download YouTube playlist videos in separate directories according to their uploaded year
+`yt-dlp -o "%(upload_date>%Y)s/%(title)s.%(ext)s" "https://www.youtube.com/playlist?list=PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re"`
```
Comment (blueskyson) [inline pages/common/yt-dlp.md:17]:
```suggestion
- Download YouTube playlist videos in separate directories according to their uploaded years:

`yt-dlp -o "%(upload_date>%Y)s/%(title)s.%(ext)s" "https://www.youtube.com/playlist?list=PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re"`
```

File: pages/common/yt-dlp.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download all playlists of YouTube channel/user keeping each playlist in separate directory:
+`yt-dlp -o "%(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s" "https://www.youtube.com/user/TheLinuxFoundation/playlists"`
```
Comment (blueskyson) [inline pages/common/yt-dlp.md:20]:
```suggestion
- Download all playlists of YouTube channel/user and keep each playlist in separate directories:

`yt-dlp -o "%(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s" "https://www.youtube.com/user/TheLinuxFoundation/playlists"`
```

File: pages/common/yt-dlp.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download Udemy course keeping each chapter in separate directory under MyVideos directory in your home
+`yt-dlp -u user -p password -P "~/MyVideos" -o "%(playlist)s/%(chapter_number)s - %(chapter)s/%(title)s.%(ext)s" "https://www.udemy.com/java-tutorial"`
```
Comment (blueskyson) [inline pages/common/yt-dlp.md:23]:
```suggestion
- Download Udemy course and keep each chapter in separate directories under "~/MyVideos":

`yt-dlp -u user -p password -P "~/MyVideos" -o "%(playlist)s/%(chapter_number)s - %(chapter)s/%(title)s.%(ext)s" "https://www.udemy.com/java-tutorial"`
```

File: pages/common/yt-dlp.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download entire series season keeping each series and each season in separate directory under C:/MyVideos
+`yt-dlp -P "C:/MyVideos" -o "%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s" "https://videomore.ru/kino_v_detalayah/5_sezon/367617"`
```
Comment (blueskyson) [inline pages/common/yt-dlp.md:26]:
```suggestion
- Download entire series season and keep each series and each season in separate directories under "C:/MyVideos":

`yt-dlp -P "C:/MyVideos" -o "%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s" "https://videomore.ru/kino_v_detalayah/5_sezon/367617"`
```

**Comment by navarroaxel** [comment]
Oh, you deleted the branch. Just a rebase interactive, commit amend and a push force you could fix the author stuff.

**Pull request closed without merging**